### PR TITLE
fix: Update content and styling of use case getting started cards

### PR DESCRIPTION
**Pull request by virajsanghvi**
### Description

Updates getting started cards:
- Updates quantity to avoid having more than will fit on a page 
- Updates content to be consistent
- Updates visual design to be consistent + use new styling

Callouts:
- This requires an oui update for new icons so please hold off on merging until the next oui bump.
- This intentionally tries to use OuiCards as they are today as we haven't designed a getting started component. All changes are only for getting started card use cases, so other cards aren't impact. I did override some oui styles as that seemed like the simplest option until we're able to create a dedicated component.

### Issues Resolved

N/A

## Screenshot

essentials
image

search
image

all
image

observability (updated in separate pr but including here for completion)
image

security analytics (updated in separate pr but including here for completion)
image

## Testing the changes

Tested locally and reviewed by ux. Updated tests that existed, but some of these don't have snapshot tests in place. Can create if desired.

## Changelog
- fix: Update content and styling of use case getting started cards

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by virajsanghvi** [comment]
Putting this out for review and approvals, but please don't merge if it has approvals (want an oui bump first so icons show up)

**Review by Hailong-am — APPROVED** [review]
thanks

**Review by TackAdam — APPROVED** [review]

**Review by ruanyl — APPROVED** [review]

**Pull request merged**
